### PR TITLE
Rooms for period logic

### PR DESCRIPTION
**Pull request by vvoinarovych**

### CODE REVIEW

File: src/main/java/com/sda/conference_room/model/entity/ConferenceRoom.java [hunk 1]
```diff
@@ -39,4 +42,6 @@ public class ConferenceRoom {
 
     @ManyToOne(cascade = CascadeType.PERSIST)
     private Organization organization;
+
```
Comment (mateuszbyczkowski) [inline src/main/java/com/sda/conference_room/model/entity/ConferenceRoom.java:45]: Unnecessary break lines.

File: src/main/java/com/sda/conference_room/rest/controller/ConferenceRoomController.java [hunk 2]
```diff
@@ -11,6 +11,7 @@
 import java.util.List;
 
 @RestController
+@CrossOrigin
```
Comment (mateuszbyczkowski) [inline src/main/java/com/sda/conference_room/rest/controller/ConferenceRoomController.java:14]: Do you need it? 
In one of classes you had
@Configuration
public class WebConfiguration implements WebMvcConfigurer {

    @Override
    public void addCorsMappings(CorsRegistry registry) {
        registry.addMapping("/**").allowedMethods("*");
    }
}

File: src/main/java/com/sda/conference_room/rest/controller/ReservationController.java [hunk 3]
```diff
@@ -22,6 +25,11 @@ public ResponseEntity<List<ReservationDto>> getAllReservations() {
         return ResponseEntity.ok(reservationService.getAllReservations());
     }
 
+    @GetMapping("/all/{organizationId}")
+    public ResponseEntity<List<ReservationDto>> getAllReservationsForOrganization(@PathVariable("organizationId") Long id){
+        return ResponseEntity.ok(reservationService.getAllReservationsByOrganizationId(id));
+    }
+
     @GetMapping("/find/{id}")
```
Comment (mateuszbyczkowski) [inline src/main/java/com/sda/conference_room/rest/controller/ReservationController.java:33]: Do not use find, update and delete words. You don't need them.

File: src/main/java/com/sda/conference_room/service/ReservationService.java [hunk 4]
```diff
@@ -1,7 +1,9 @@
 package com.sda.conference_room.service;
 
+import com.sda.conference_room.model.dto.ConferenceRoomDto;
 import com.sda.conference_room.model.dto.ReservationDto;
 
+import java.time.LocalDateTime;
 import java.util.List;
 
 public interface ReservationService {
```
Comment (mateuszbyczkowski) [inline src/main/java/com/sda/conference_room/service/ReservationService.java:9]: Do you need this interface?

**Review by lokrogi — APPROVED** [review]

**Pull request merged**